scripts/hudson/narayana.sh

```diff
@@ -411,20 +379,21 @@ function clone_as {
 }
 
 function build_as {
-  echo "Building JBoss EAP/WildFly"
+  AS_BRANCH=27.0.0.Alpha1
+  echo "Building WildFly $AS_BRANCH"
 
-  cd $WORKSPACE/jboss-as
+  cd jboss-as
```

Comment:
I reckon this could still say `cd $WORKSPACE/jboss-as` to minimize the change set